web-ng/src/app/components/option-editor/option-editor.component.ts

```diff
@@ -56,6 +59,9 @@ export class OptionEditorComponent implements OnInit, OnChanges {
   }
 
   ngOnChanges() {
+    if (this.optionInput) {
```

Comment:
This causes the focus to be placed into the option label _on every change_ to anything on the option form (including Code). Instead, we just want to focus here when a new option is added.